.github/workflows/test.yml

```diff
@@ -12,51 +12,22 @@ on:
     types: [ released ]
 
 jobs:
-  flake8:
-    name: flake8
+  linting:
     runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-      - run: python -m pip install flake8
-      - name: flake8
-        uses: liskin/gh-problem-matcher-wrap@v1
-        with:
-          linters: flake8
-          run: flake8
-
-  isort:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-      - run: python -m pip install isort
-      - name: isort
-        uses: liskin/gh-problem-matcher-wrap@v1
-        with:
-          linters: isort
-          run: isort --check --profile black src/
+    strategy:
+      matrix:
+        python-version: ["3.11"]
```

Comment:
Why change it to python 3.11? We haven't tested its support yet

Reply:
Sorry this was not intended